modules/api-svc/src/lib/model/lib/shared.js

```diff
@@ -428,6 +471,61 @@ const internalBulkQuotesResponseToMojaloop = (internal) => {
     return external;
 };
 
+/**
+ * Converts an internal bulk transfer response to mojaloop POST /bulkTransfers/{ID} response
+ *
+ * @returns {object}
+ */
+const internalBulkTransfersResponseToMojaloop = (internal, fulfilments) => {
+    const external = {
+        completedTimestamp: new Date(),
+        bulkTransferState: FSPIOPBulkTransferStateEnum.COMPLETED,
+    };
+
+    if(internal.individualTransferResults && internal.individualTransferResults.length) {
+        const individualTransferResults = internal.individualTransferResults.map((transfer) => {
+            let externalTransfer;
+            // How do we map fsp backend errors to mojaloop errors?
+            // For now we will map any error returned by the backend as a
+            // mojaloop payee rejection and put the fsp backend error into an extension
+            // list.
+            if(transfer.errorResponse) {
+                externalTransfer = {
+                    transferId: transfer.transferId,
+                    errorInformation: Errors.MojaloopApiErrorObjectFromCode(Errors.MojaloopApiErrorCodes.PAYEE_REJECTION).errorInformation,
+                    extensionList: {
+                        extension: [
+                            {
+                                key: 'errorResponseStatusCode',
+                                value: transfer.errorResponse.statusCode
+                            },
+                            {
+                                key: 'errorResponseMessage',
```

Comment:
I assume this was a mapping that already existed?

Reply:
Added in https://github.com/mojaloop/sdk-scheme-adapter/pull/404